commit 0b33e4c9ff85b8653fee0d5503242322119f25ed
Author: Emmanuel Lepage Vallee <elv1313@gmail.com>
Date:   Sat Nov 1 18:53:03 2014 -0400

    rc.lua: Prevent clients from being lost on RANRD changes

--- a/awesomerc.lua.in
+++ b/awesomerc.lua.in
@@ -396,6 +396,9 @@
             awful.placement.no_overlap(c)
             awful.placement.no_offscreen(c)
         end
+    elseif not c.size_hints.user_position and not c.size_hints.program_position then
+        -- Prevent clients from being unreachable after screen count change
+        awful.placement.no_offscreen(c)
     end
 
     local titlebars_enabled = false
